specification/applicationinsights/resource-manager/Microsoft.Insights/preview/2018-05-01/componentProactiveDetection_API.json

```diff
@@ -0,0 +1,315 @@
+{
```

Comment:
This file is never referenced from the readme file. So, there is no validation running on it. Please add a suitable tag and update this file in the readme file. So, I can review the same.

Reply:
sorry, can you please explain where should I reference this file exactly? 

---
In reply to: [271035135](https://github.com/Azure/azure-rest-api-specs/pull/5518#discussion_r271035135) [](ancestors = 271035135)

Reply:
Refer example here: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/applicationinsights/resource-manager/readme.md#tag-package-2018-06-17-preview

Reply:
Done.

---
In reply to: [271504122](https://github.com/Azure/azure-rest-api-specs/pull/5518#discussion_r271504122) [](ancestors = 271504122)